packages/bits/src/lib/wizard-v2/wizard.directive.ts

```diff
@@ -116,6 +135,22 @@ export class WizardDirective extends CdkStepper implements OnChanges, AfterConte
         super.ngOnDestroy();
     }
 
+    public getStepState(i: number, state: StepState = STEP_STATE.NUMBER): StepState {
```

Comment:
I'm not sure it makes sense to have a state argument here. Can we just return `STEP_STATE.NUMBER` regardless at the bottom of the method?